### ISSUE TITLE
Daemon engine does not currently support ppc64le arch

### ISSUE THREAD

**Issue by anengineer1** [DaemonEngine/Daemon#226]
Greetings, the engine does not currently support ppc64le architecture and its popularity is getting increased lately.

**Comment by illwieckz** [comment]
Hi, this is known, see https://github.com/Unvanquished/Unvanquished/issues/1086
In any way, we'll keep open your issue as the engine-side one.

There is two topics to explore:

- getting rid of potential x86ism
- porting nacl to wasm

Feel free to apply on anything you may think you can be helpful, at least identifying properly what would be missing or what would be broken is helpful :-)

It would be very cool to see Unvanquished and Dæmon engine running on Power

**Comment by illwieckz** [comment]
I created #227 for the Wasm thing, this one will be the parent issue for tracking x86ism.

**Comment by anengineer1** [comment]
Thank you a lot for your kind reply!

If I find myself in a position to help some way I certainly will.

Yes, it would be so sweet indeed to see Daemon engine under Power!

Thank you again for everything

**Comment by illwieckz** [comment]
A position of tester would be helpful, do you have access to such hardware?

**Comment by anengineer1** [comment]
Yes, I have a ppc64le and the fedora os installed. Just tell me, and I will do the testing!!! :-)

I'm at you service!

**Comment by bmorel** [comment]
I'm not sure if this is a bug or a feature request... lack of portability seems a bug, to me, but that's also quite close to a feature request...

**Comment by slipher** [comment]
Fixed by #1945, #1946.
